test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -7896,5 +7896,124 @@ public async Task RestoreNetCore_PackagesLockFile_WithProjectAndPackageReference
                 r.Success.Should().BeTrue();
             }
         }
+
+        [Fact]
+        public async Task RestoreNetCore_ExclusiveLowerBound_RestoreSucceeds()
```

Comment:
I assume that the test should have restored the 1.1.0 package and not the 1.0.0 package, right? If not, I don't understand the point of creating two packages, just one would be sufficient, right? Is it worthwhile adding an assert to make it clear to the reader what's expected?

Same for the other test.

Reply:
I can add an assert to verify that 1.1.0 was restored. 

However, both packages are needed. This is only a problem when the exclusively lower bound is actually found on the feed.

Reply:
Improved the test.